<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Driver</class>
 <widget class="QWidget" name="Driver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>481</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="main_page">
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>120</y>
       <width>291</width>
       <height>181</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <bold>true</bold>
              <underline>true</underline>
             </font>
            </property>
            <property name="text">
             <string>Добро пожаловать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <italic>true</italic>
             </font>
            </property>
            <property name="text">
             <string>Выберите действие</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="activOrder_button">
          <property name="text">
           <string>Просмотр активного заказа</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="listDoneOrder_button">
          <property name="text">
           <string>Список выполненных заказов</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit">
          <property name="text">
           <string>Выход</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="list_page">
    <widget class="QPushButton" name="back_button_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>270</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="activOrder_page">
    <widget class="QPushButton" name="accept_button">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>250</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Подтвердить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="back_button">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>250</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
